### ISSUE TITLE
The space disappears after converting a negative number

### ISSUE THREAD

**Issue by sgromkov** [Lakkanna/css-converter#21]
Original CSS:
```css
background-position: 0 -10px;
```

Current conversion result:
```jsx
backgroundPosition: "0-10px",
```

Expected result:
```jsx
backgroundPosition: "0 -10px",
```

**Comment by Lakkanna** [comment]
Hi @sgromkov 
I'm sorry for the delayed response on this issue. The issue has been fixed in version 0.2.0 now.

Original CSS:

```
background-position: 0 -10px;
```

Current conversion result:
```
backgroundPosition: "0 -10px",
```

**Comment by Lakkanna** [comment]
Hi @sgromkov,

I'm glad to inform you that the issue you reported has been fixed in the latest version. We've closed this issue for now, but if you find that the issue still persists in the latest version, please feel free to reopen this issue and provide us with more details.

Thank you for bringing this issue to our attention and helping us to improve extension.
